resources/web/data/text.js

```diff
@@ -1111,11 +1111,11 @@ var LangText = {
     t111: "Создать новый",
     t112: "Присоединяйтесь к программе",
     t113: "Вы можете изменить свой выбор в любое время.",
-    t126: "Загрузка идёт……",
+    t126: "Загрузка……",
     orca1: "Редактировать информацию о проекте",
     orca2: "Информации о модели отсутствует",
     orca3: "Режим конфиденциальности",
-    orca4: "Это остановит передачу данных в облачные сервисы Bambu. Пользователи, которые не используют принтеры Bambu Lab или используют режим «Только LAN», могут безопасно включить эту функцию.",
+    orca4: "Это остановит передачу данных в облачные сервисы Bambu. Пользователи, которые не используют принтеры Bambu Lab или используют их офлайн, могут спокойно включать эту опцию.",

```

Comment:
Такой ещё есть вариант:
```suggestion
    orca4: "Это остановит передачу данных в облачные сервисы Bambu. Пользователи, которые не используют принтеры Bambu Lab или используют их локально, могут спокойно включать эту опцию.",
```

Reply:
Насчет text.js я практики все принял, поэтому просто заменил его, чтобы тут не тыкать. Неудобно все же в браузере 😠

Кроме последнего...

— Это остановит передачу данных в облачные сервисы Bambu. Пользователи, которые не используют принтеры Bambu Lab или используют их в режиме «Только LAN», могут безопасно включить эту опцию.

Просто там пункт в меню принтера на английском так и называется «LAN only», и чтобы людей не путать, и он сразу, так сказать, в глаза бросался и сопоставлялся с русским переводом, поэтому так и писал везде. Опять же, не все понимают, что такое офлайн, некоторые будут думать, что и работа по локальной сеть нафиг отрубится при включении ползунка.

И вот потом надо глянуть, как его перевели в официальном переводе на модели A1 с учетом, что там маленький экран и надо впихнуть невпихуемое, не как на других моделях (где и интерфейс другой)... хотя хз.
<img width="582" height="424" alt="image" src="https://github.com/user-attachments/assets/735ec527-217a-4044-8353-5144dacf69de" />

Reply:
Да, я поэтому в этом комментарии предложил ещё общий вариант с "используют принтер локально". Ещё есть такой инвертированный вариант:
```suggestion
    orca4: "Это остановит передачу данных в облачные сервисы Bambu. Помешает только владельцам Bambu Lab, не использующим режим «Только LAN».",
```